compiler/rustc_mir_build/src/build/matches/mod.rs

```diff
@@ -1698,13 +1711,14 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
 // Pat binding - used for `let` and function parameters as well.
 
 impl<'a, 'tcx> Builder<'a, 'tcx> {
-    pub fn lower_let(
+    crate fn lower_let_else(
```

Comment:
This confused me cause I just added `lower_let_else` in AST lowering for the new language feature. Maybe this should be left as `lower_let`? But no big deal.